### ISSUE TITLE
Convert OMR Acceptance build to use Pipeline-Build-Test-All

### ISSUE THREAD

**Issue by AdamBrousseau** [eclipse/openj9#2778]
Related #2728 #1897 #2272 #2160 

The [OMR-Acceptance build](https://ci.eclipse.org/openj9/job/Pipeline-OMR-Acceptance) currently uses its own top level [pipeline file](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/jobs/pipelines/Pipeline-OMR-Acceptance) that calls Build and Test jobs directly ([workflow function](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/pipeline-functions#L237)). This is inconsistent with how the other builds work and has proven to be a pain point as we make changes to the pipelines (eg. #2272). [Pipeline-Build-Test-All](https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All) was introduced as a reusable top level "build what you want" file. We should reuse this file rather than maintaining two separate files that do somewhat the same thing.

The most notable differences will be:
- OMR build will now be 3 levels deep like the nightly build (currently only 2 levels)
   - i.e OMR -> Pipeline-B-T-JDKX-SPEC -> Build-JDKX-SPEC
- Slack notifications will no longer indicate the build is OMR
- Current:
```
Failure in: Pipeline-OMR-Acceptance #731 (Open)
Downstream Job: Test-sanity.functional-JDK8-win_x86-64_cmprssptrs #73 (Open)
```
   - New: 
```
Failure in: Pipeline-Build-Test-JDK8-win_x86-64_cmprssptrs #20 (Open)
Downstream Job: Test-sanity.functional-JDK8-win_x86-64_cmprssptrs #73 (Open)
```
Note: We could add something to the output to indicate upstream job name.

Other considerations:
- The current build has a hack in order to run the cmake spec (compile only). This would have to be investigated on how to add it to the new style build. (cc @dnakamura )
- This would also add win32 (jdk8 only) to the build which is not currently added. The logic in order to only run it on 8 exists in Pipe-B-T-All but hasn't been copied into the current OMR pipe file. Converting the build would eliminate the need to copy this logic.

Changes Required:
- The OMR build would need to point to the P-B-T-All pipeline file instead of the current one
- The OMR Pipe can be deleted
- Add Params to the OMR Build
   - OMR_BRANCH = master
   - Java8 = true
   - Java10 = true
   - TESTS_TARGETS = sanity.functional
- Some logic would need to be added to the end of P-B-T-All in order to fire the Promotion at the end of the build.

cc @jdekonin @vsebe

**Comment by AdamBrousseau** [comment]
My test build works with the changes from PR #2780 
https://ci.eclipse.org/openj9/view/Pipelines/job/Pipeline-OMR-Acceptance-New/3/

**Comment by AdamBrousseau** [comment]
resolved via #2780
